core/src/Authorization.html

```diff
@@ -81,23 +81,42 @@
         isHidden: false
       };
     },
-    oncreate() {
+    async oncreate() {
       if (!LuigiConfig.isAuthorizationEnabled()) {
```

Comment:
You should use the isAuthorizationEnabled() from auth (not config, as this is deprecated now).